packages/compass-aggregations/src/components/aggregation-side-panel/stage-wizard-use-cases/lookup/lookup.tsx

```diff
@@ -0,0 +1,191 @@
+import {
+  Body,
+  spacing,
+  css,
+  TextInput,
+  Combobox,
+  ComboboxWithCustomOption,
+  ComboboxOption,
+} from '@mongodb-js/compass-components';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { connect } from 'react-redux';
+import type { RootState } from '../../../../modules';
+import { fetchCollectionFields } from '../../../../modules/collections-fields';
+import type { CollectionData } from '../../../../modules/collections-fields';
+
+type LookupFormState = {
+  from: string;
+  localField: string;
+  foreignField: string;
+  as: string;
+};
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[2],
+});
+
+const titleStyles = css({
+  minWidth: `${'Join documents in'.length}ch`,
```

Comment:
I'm thinking if we to store this `Join documents in` string as a variable and reference it in both places we can prevent styling issues down the line if the text changes and someone misses this style.